fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/electric/providers/ChargingStationIndex.java

```diff
@@ -49,14 +49,15 @@ public class ChargingStationIndex {
 
     public ChargingStationIndex(int bucketSize) {
         this.bucketSize = bucketSize;
```

Comment:
I really wonder if we want to make the bucket-size configurable.. We hard-code it anyhow when we initialize ChargingStationIndex, don't we?